Makefile

```diff
@@ -100,3 +100,10 @@ sidecar: sidecar-wp-cli
 sidecar-wp-cli:
 	docker build -t $(SIDECAR_WP_CLI_IMG) -f $(SIDECAR_DIR)/wp-cli.dockerfile $(SIDECAR_DIR)
 	docker push $(SIDECAR_WP_CLI_IMG)
+
+SYSBENCH_IMG=$(DOCKER_HUB_REPO)/sysbench:latest
```

Comment:
this should run from the default torpedo make rules

Reply:
it's already using DOCKER_HUB_REPO which is used all across Makefile